app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,65 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      def show
+        volunteer = Volunteer.find(params[:id])
+        render json: { status: 'SUCCESS', message: 'Loaded volunteer', data: volunteer }, status: :ok
+      end
+
+      def create
+        volunteer = Volunteer.new(volunteer_params)
+
+        if volunteer.save
+          render json: { status: 'SUCCESS', message: 'Created volunteer', data: volunteer }, status: :ok
+        else
+          render json: { status: 'ERROR', message: 'Volunteer not saved', data: volunteer.errors }, status: :unprocessable_entity
+        end
+      end
+
+      def update
+        volunteer = Volunteer.find(params[:id])
+
+        if volunteer.update(volunteer_params)
+          render json: { status: 'SUCCESS', message: 'Updated volunteer', data: volunteer }, status: :ok
+        else
+          render json: { status: 'ERROR', message: 'Volunteer not updated', data: volunteer.errors }, status: :unprocessable_entity
+        end
+      end
+
+      def destroy
+        volunteer = Volunteer.find(params[:id])
+        volunteer.destroy
+
+        render json: { status: 'SUCCESS', message: 'Deleted volunteer', data: volunteer }, status: :ok
```

Comment:
Delete actions should render no content. Set the `head: :no_content`. The render method isn't necessary here as no json is rendered/expected. Move the find statement into a private method as described above.

Reply:
Done